cultcargo/sunblocker.yml

```diff
@@ -0,0 +1,148 @@
+_include:
+  - genesis/cult-cargo-base.yml
+
+cabs:
+    sunblocker:
+        info: A pyrap method to remove solar-like RFI from interferometric data (https://github.com/gigjozsa/sunblocker)
+        image:
+            _use: vars.cult-cargo.images
+            name: sunblocker
+        flavour:
+            kind: python-code
+            input_vars: false
+            input_dict: options
+        command: |
+            import subprocess
```

Comment:
No need for import, since it's commented out.

Reply:
duly